README.md

```diff
@@ -1,10 +1,6 @@
 # What is `kedro-examples`?
 
-This repo contains example code to help you learn how to use [Kedro](https://github.com/quantumblacklabs/kedro).
-
-## How do I use it?
-
-Each example has an README.md that explains what each project does and how to work with it.
+This repo contains links to all official and community created example Kedro projects as well as the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).
```

Comment:
```suggestion
This repo contains links to all official and community created example Kedro projects as well as the complete version of the [spaceflights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).
```

In the Kedro docs we refer to it as "spaceflights" one word, and typically don't capitalise it.